.github/lyft_maintainers.yml

```diff
@@ -0,0 +1,7 @@
+current: jpsim
```

Comment:
Maybe we can fold the maintainer role into this and we can better set expectations with our friends who want to be part of this -- wdyt?

Reply:
I'm not sure I understand what you mean. Can you elaborate on what you mean by folding this maintainer role into this? This file _does_ define who the current support maintainer is.